port/common/j9nls.c

```diff
@@ -1215,11 +1215,11 @@ read_from_catalog(struct OMRPortLibrary *portLibrary, intptr_t fd, char *buf, in
 	char temp[BUF_SIZE];
 	intptr_t count, nbytes = bufsize;
 	char *cursor = buf;
-#ifdef J9ZOS390
+#ifdef defined(J9ZOS390) && !defined(OMR_EBCDIC)
```

Comment:
This (and other places like it) must be `#if ...`, not `#ifdef`; see the build errors in https://ci.appveyor.com/project/eclipsewebmaster/omr/builds/26352320

Reply:
Fixed (see [force push](https://github.com/eclipse/omr/compare/3e61353e14cfd0e19303d2faaa8d0d3b8e0f636b..3810be5b2a9a6cdbb08e38f8daac41fe703e4311)).